lib/test/helpers/util/wait.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+/* jshint ignore:start */
+/**
+ * Asynchronously wait for a specified number of milliseconds.
+ *
+ * @param {number} ms - Number of milliseconds to wait.
+ *
+ * @return {Promise<void>} The promise to await on.
+ */
+async function wait(ms) {
```

Comment:
I would move it to `lib/utils` ’coz it looks like not just trsts related function.

Reply:
??? didn't catch. why this file should be outside 'test' folder?

Reply:
Nevermind. Just forget about it.